### PR TITLE
Update Lisk SDK to v6.0.0-alpha.16

### LINKED ISSUE #742: Update Lisk SDK to v6.0.0-alpha.16
### Description

Update Lisk SDK to v6.0.0-alpha.16

### Acceptance Criteria

- Update Lisk SDK to v6.0.0-alpha.16
- Address upstream interface changes, if any
  - Update the config files
  - Update the asset files
- Update the following commands:
  - Remove `src/commands/blskey`
  - Add `src/commands/config/create.ts`
  - Rename `src/commands/node` to `src/commands/system`
- Update `$id` for all schemas in `src/application/modules/legacy/schemas.ts` to match the definition patterns used in SDK (Ref: `framework/src/modules/pos/schemas.ts`)
- Regenerate the `devnet` and `alphanet` genesis blocks
- Ensure Lisk Core builds and is able to start
- Able to create/post all types of transactions including cross-chain transfers

### PR DESCRIPTION
**Pull request by sameersubudhi**
### What was the problem?

This PR resolves #742 #744 

### How was it solved?

- [x] Updated Lisk SDK to v6.0.0-alpha.16
  - Generated the genesis blocks for both devnet and alphanet
  - Minor config refactoring for both devnet and alphanet
- [x] Added support for `config:create` command
- [x] Removed unused files (`src/commands/blskey/create.ts`)
- [x] Updated README
- [x] Updated `$id`(s) for all legacy module schemas to match pattern followed by SDK 

### How was it tested?

Local
 - Run stress test
 - Make cross-chain registration
 - Make cross-chain token transfer transaction (untested, needs proper setup... but was able to successfully send CCMs from sidechain to mainchain)

### CODE REVIEW

File: src/commands/blskey/create.ts [hunk 1]
```diff
@@ -1 +0,0 @@
-export { KeysCreateCommand } from 'lisk-commander';
```
Comment (ManuGowda) [inline src/commands/blskey/create.ts:1]: why did we remove this?

**Review by ManuGowda — APPROVED** [review]

File: src/commands/blskey/create.ts (hunk 1 above)
Comment (sameersubudhi) [reply src/commands/blskey/create.ts:1]: This didn't work anymore. SDK has also removed the support for the same.

File: docs/commands/node.md [hunk 2]
```diff
@@ -19,8 +21,29 @@ OPTIONS
...
+
+OPTIONS
+  -d, --data-path=data-path  Directory path to specify where node data is stored. Environment variable "LISK_DATA_PATH"
+                             can also be used.
+
+  --pretty                   Prints JSON in pretty format rather than condensed.
+
+EXAMPLES
+  system:metadata
+  system:metadata --data-path ./lisk
```
Comment (nagdahimanshu) [inline docs/commands/node.md:46]:
```suggestion
  node:metadata
  node:metadata --data-path ./lisk
```

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Pull request merged**